packages/react/src/utils/useScrollLock.ts

```diff
@@ -94,11 +94,17 @@ function preventScrollStandard(referenceElement?: Element | null) {
     // Handle `scrollbar-gutter` in Chrome when there is no scrollable content.
     const hasScrollbarGutterStable = htmlComputedStyles.scrollbarGutter?.includes('stable');
 
+    // Safari needs visual viewport offsets added to account for pinch-zoom
+    const webkit = isWebKit();
+    const vV = doc.defaultView?.visualViewport;
```

Comment:
I'm curious: is `defaultView` different to `window`?

Reply:
It's the owning document's window

Reply:
:+1:
We have a utility to return this - `ownerWindow` in `utils/owner`